src/control/server/commands.go

```diff
@@ -33,20 +33,20 @@ import (
 
 // cliOptions struct defined flags that can be used when invoking daos_server.
 type cliOptions struct {
-	Port       uint16  `short:"p" long:"port" description:"Port for the gRPC management interfect to listen on"`
-	MountPath  string  `short:"s" long:"storage" description:"Storage path"`
-	ConfigPath string  `short:"o" long:"config_path" description:"Server config file path"`
-	Modules    *string `short:"m" long:"modules" description:"List of server modules to load"`
-	Cores      uint16  `short:"c" long:"cores" default:"0" description:"number of cores to use (default all)"`
-	Targets    int     `short:"t" long:"targets" default:"0" description:"number of targets to use (default use all cores)"`
-	XShelpernr int     `short:"x" long:"xshelpernr" default:"2" description:"number of helper XS per VOS target (default 2)"`
-	Firstcore  int     `short:"f" long:"firstcore" default:"0" description:"index of first core for service thread (default 0)"`
-	Group      string  `short:"g" long:"group" description:"Server group name"`
-	Attach     *string `short:"a" long:"attach_info" description:"Attach info patch (to support non-PMIx client, default /tmp)"`
-	Map        *string `short:"y" long:"map" description:"[Temporary] System map file"`
-	Rank       *rank   `short:"r" long:"rank" description:"[Temporary] Self rank"`
-	SocketDir  string  `short:"d" long:"socket_dir" description:"Location for all daos_server & daos_io_server sockets"`
-	Storage    StorCmd `command:"storage" alias:"st" description:"Perform tasks related to locally-attached storage"`
+	Port        uint16  `short:"p" long:"port" description:"Port for the gRPC management interfect to listen on"`
+	MountPath   string  `short:"s" long:"storage" description:"Storage path"`
+	ConfigPath  string  `short:"o" long:"config_path" description:"Server config file path"`
+	Modules     *string `short:"m" long:"modules" description:"List of server modules to load"`
+	Cores       uint16  `short:"c" long:"cores" default:"0" description:"option deprecated, please use targets instead"`
+	Targets     uint16  `short:"t" long:"targets" default:"0" description:"number of targets to use (default use all cores)"`
```

Comment:
I'm not up to speed on the rationale of changing from 'cores' to 'targets'.  However, the new help text mixes 'targets' and 'cores' and it's not clear why.  It says ' ... (default use all cores) '.  It seems like it should say ' ... (default use all targets)'.

Reply:
targets run on cores, this was xuezhao's wording and seems okay to me. it's basically saying that by default you have as many targets as you have cores.

Reply:
-t configures the number of vos target, each vos tgt can with multiple XS thread (-x number + 1).
the original -c is not really reasonable as it is not number of cores.